core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala

```diff
@@ -89,7 +89,7 @@ trait RawlsApiService //(val workspaceServiceConstructor: UserInfo => WorkspaceS
   implicit val executionContext: ExecutionContext
   implicit val materializer: Materializer
 
-  val baseApiRoutes = workspaceRoutes ~ entityRoutes ~ methodConfigRoutes ~ submissionRoutes ~ adminRoutes ~ userRoutes ~ billingRoutes ~ billingRoutesV2 ~ notificationsRoutes ~ servicePerimeterRoutes ~ snapshotRoutes
+  val baseApiRoutes = workspaceRoutes ~ entityRoutes ~ methodConfigRoutes ~ submissionRoutes ~ adminRoutes ~ userRoutes ~ billingRoutesV2 ~ billingRoutes ~ notificationsRoutes ~ servicePerimeterRoutes ~ snapshotRoutes
```

Comment:
This change is because the order that the routes were concatenated was causing some ambiguity and made these 2 new endpoints unreachable. 

`/api/billing/v2/{projectId}/spendReportConfiguration`
overlaps with
`/api/billing/{projectId}/{workbenchRole}/{email}`

for both `PUT` and `DELETE`.

The side effect is that calling the latter won't work if your project name is `v2`, but given that you'd be forced to call it with an invalid `workbenchRole` and `email`, I don't see that as a problem. I don't think that v2 is even a valid project name either.